docs/admin/region-availability.md

```diff
@@ -34,7 +37,65 @@ For more information about purchasing a Fabric subscription, see [Buy a Microsof
 | West US 3         | UK South            |             |                    |                    |
 |                   | UK West             |             |                    |                    |
 
+# Power BI F SKU Regional Availability
+
+New regions or constrained regions will   only support Power BI artifacts. Over time, the new regions will expand to include additional Fabric artifacts. Currently, the regions listed below support Power BI and are in the process of onboarding additional Fabric artifacts. Note that India West is capacity constrained and will exclusively host Power BI on the F SKU and not launch other Fabric features.
```

Comment:
Something doesn't add up. In the overview new/constrained regions didn't support 'Fabric features' and here's it's Power BI...

In any case, this is just marketing speak. Can we remove this or make it factual?

Reply:
This one makes sense to me. The first table has a list of regions where all the Fabric items are available. This second list of regions don't have all the Fabric items, but at a minimum do have all Power BI items. In the future engineering will ship the full experience to the regions in the second list and those regions will then move to the first list.